exec/java-exec/src/main/java/org/apache/drill/exec/vector/complex/fn/JsonReader.java

```diff
@@ -110,21 +118,29 @@ public void ensureAtLeastOneField(ComplexWriter writer) {
         emptyStatus.set(i, true);
       }
       if (i == 0 && !allTextMode) {
-        // when allTextMode is false, there is not much benefit to producing all the empty
-        // fields; just produce 1 field.  The reason is that the type of the fields is
-        // unknown, so if we produce multiple Integer fields by default, a subsequent batch
-        // that contains non-integer fields will error out in any case.  Whereas, with
-        // allTextMode true, we are sure that all fields are going to be treated as varchar,
-        // so it makes sense to produce all the fields, and in fact is necessary in order to
+        // when allTextMode is false, there is not much benefit to producing all
```

Comment:
Seems the line width changed here(line 121-132). Please reorganize the text. If possible keep the original text unchanged. The same in line 140-143